improver_tests/nbhood/nbhood/test_NeighbourhoodProcessing.py

```diff
@@ -50,11 +50,7 @@ def test_neighbourhood_method_exists(self):
          method exists."""
         neighbourhood_method = "circular"
         radii = 10000
-        result = NBHood(neighbourhood_method, radii)
```

Comment:
As above 

This can't be tested any more, but the test is designed to make sure the init doesn't raise an error. The test will still fail if it does start raising an error. Otherwise I could get rid of this test entirely.